src/VisualStudio/Core/Def/Implementation/FindReferences/FindUsagesValueUsageInfoColumnDefinition.cs

```diff
@@ -33,6 +35,24 @@ public FindUsagesValueUsageInfoColumnDefinition()
         {
         }
 
+        public override bool TryGetFilterItems(ITableEntryHandle entry, out IEnumerable<string> filterItems)
```

Comment:
Shouldn't this override be in `AbstractCustomColumnDefinitionWithMultipleValues`?